client/my-sites/customer-home/main.jsx

```diff
@@ -157,51 +163,98 @@ class Home extends Component {
 							{ translate( 'Review and update my site' ) }
 						</h6>
 						<div className="customer-home__card-button-pair">
-							<Button href={ site.URL } primary>
+							<Button
+								href={ site.URL }
+								primary
+								onClick={ () => trackAction( 'my_site', 'view_site' ) }
+							>
 								{ translate( 'View Site' ) }
 							</Button>
-							<Button href={ customizeUrl }>{ translate( 'Edit Homepage' ) }</Button>
+							{ isStaticHomePage ? (
+								<Button
+									href={ customizeUrl }
+									onClick={ () => trackAction( 'my_site', 'edit_homepage' ) }
+								>
+									{ translate( 'Edit Homepage' ) }
+								</Button>
+							) : (
+								<Button
+									onClick={ () => {
+										trackAction( 'my_site', 'write_post' );
+										page( `/post/${ siteSlug }` );
+									} }
+								>
+									{ translate( 'Write Blog Post' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Write blog post' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/customer-home-analytics)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).